dataingest/spatial/import.sh

```diff
@@ -2,21 +2,14 @@
 
 set -e
 
```

Comment:
worth adding this version check / prompt for bash, for the benefit of mac users?
`[ "${BASH_VERSINFO:-0}" -lt 4 ] && echo "bash 4 or higher needed, try brew install bash (on MacOS) to update" && exit 1`

Reply:
I changed the script to avoid the modern associative arrays for compatibility with mac's old bash.